pupil-api/src/services/feature.service.ts

```diff
@@ -24,29 +23,7 @@ export class FeatureService implements IFeatureService {
      * Load feature toggles
      */
     logger.info('ENVIRONMENT_NAME : ' + config.Environment)
-    const environmentName = config.Environment
-    let featureTogglesSpecific = ''
-    let featureTogglesDefault = ''
-    let featureTogglesSpecificPath = ''
-    let featureTogglesDefaultPath = ''
-    try {
-      featureTogglesSpecificPath = '../feature-toggles/feature-toggles.' + environmentName
-      featureTogglesSpecific = environmentName ? require(featureTogglesSpecificPath) : null
-    // tslint:disable-next-line: no-empty
-    } catch (ignore) {} // missing feature files throw intentionally
-
-    try {
-      featureTogglesDefaultPath = '../feature-toggles/feature-toggles.default'
-      featureTogglesDefault = require(featureTogglesDefaultPath)
-    // tslint:disable-next-line: no-empty
-    } catch (ignore) {} // missing feature files throw intentionally
-
-    const featureTogglesMerged = R.mergeRight(featureTogglesDefault, featureTogglesSpecific)
-
-    if (featureTogglesMerged) {
-      logger.info(`Loading merged feature toggles from '${featureTogglesSpecificPath}', '${featureTogglesDefaultPath}': `, featureTogglesMerged)
-      featureToggles.load(featureTogglesMerged)
-    }
+    featureToggles.load(config.FeatureToggles)
```

Comment:
Can we have a line from `logger.info` saying what feature toggles are loaded please?  

E.g. 
```
logger.info(`Loading feature toggles from: `, config.FeatureToggles)
```

Reply:
good idea. added to both.